### PR TITLE
Adding PlexAniSync & Docs

### LINKED ISSUE #1224: Plexanisync
Might be a little harder then my previous request due to it needing auth from anilist (they use an API token) and a plex token. Syncs any anime played in plex to anilist (anilist.co)

Github: https://github.com/RickDB/PlexAniSync

Dockerhub: https://hub.docker.com/r/rickdb/plexanisync

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1224

### CODE REVIEW

**Comment by nemchik** [comment]
Aarch64 can use the armv7l image, but it needs a file in the compose template folder.

File: compose/.apps/plexanisync/plexanisync.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+services:
+  plexanisync:
+    command: /runsync.sh
+    container_name: plexanisync
+    environment:
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.yml:5]: Sort these please.

File: compose/.apps/plexanisync/plexanisync.labels.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    labels:
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.description: "Sync anime watched status from Plex to AniList"
+      com.dockstarter.appinfo.nicename: "Plexanisync"
+      com.dockstarter.appvars.plexanisync_enabled: "false"
+      com.dockstarter.appvars.plexanisync_network_mode: ""
+      com.dockstarter.appvars.plexanisync_plex_section: "Anime"
+      com.dockstarter.appvars.plexanisync_plex_url: "http://plex:32400"
+      com.dockstarter.appvars.plexanisync_plex_token: ""
+      com.dockstarter.appvars.plex_anisync_ani_username: ""
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.labels.yml:12]: Underscore on this one, but not the rest?

**Review by nemchik — CHANGES_REQUESTED** [review]

File: compose/.apps/plexanisync/plexanisync.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/plexanisync/plexanisync.yml:5]: I was trying to put them alphabetically, do you want `command` under `container_name`?

File: compose/.apps/plexanisync/plexanisync.labels.yml (hunk 2 above)
Comment (angrycuban13) [reply compose/.apps/plexanisync/plexanisync.labels.yml]: Fixed on 0e3231d

File: compose/.apps/plexanisync/plexanisync.yml (hunk 1 above)
Comment (nemchik) [reply compose/.apps/plexanisync/plexanisync.yml:5]: The environment variables.

File: compose/.apps/plexanisync/plexanisync.labels.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      com.dockstarter.appinfo.description: "Sync anime watched status from Plex to AniList"
+      com.dockstarter.appinfo.nicename: "Plexanisync"
+      com.dockstarter.appvars.plexanisync_enabled: "false"
+      com.dockstarter.appvars.plexanisync_network_mode: ""
+      com.dockstarter.appvars.plexanisync_plex_section: "Anime"
+      com.dockstarter.appvars.plexanisync_plex_url: "http://plex:32400"
+      com.dockstarter.appvars.plexanisync_plex_token: ""
+      com.dockstarter.appvars.plexanisync_ani_username: ""
+      com.dockstarter.appvars.plexanisync_ani_token: ""
+      com.dockstarter.appvars.plexanisync_interval: "3600"
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.labels.yml:14]:
```suggestion
      com.dockstarter.appinfo.deprecated: "false"
      com.dockstarter.appinfo.description: "Sync anime watched status from Plex to AniList"
      com.dockstarter.appinfo.nicename: "Plexanisync"
      com.dockstarter.appvars.plexanisync_ani_token: ""
      com.dockstarter.appvars.plexanisync_ani_username: ""
      com.dockstarter.appvars.plexanisync_enabled: "false"
      com.dockstarter.appvars.plexanisync_interval: "3600"
      com.dockstarter.appvars.plexanisync_network_mode: ""
      com.dockstarter.appvars.plexanisync_plex_section: "Anime"
      com.dockstarter.appvars.plexanisync_plex_token: ""
      com.dockstarter.appvars.plexanisync_plex_url: "http://plex:32400"
```
Sorted

File: compose/.apps/plexanisync/plexanisync.x86_64.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+services:
+  plexanisync:
+    image: rickdb/plexanisync:latest
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.x86_64.yml:3]:
```suggestion
    image: rickdb/plexanisync
```

File: compose/.apps/plexanisync/plexanisync.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - PLEX_SECTION=${PLEXANISYNC_PLEX_SECTION}
+      - PLEX_URL=${PLEXANISYNC_PLEX_URL}
+      - PLEX_TOKEN=${PLEXANISYNC_PLEX_TOKEN}
+      - ANI_USERNAME=${PLEX_ANISYNC_ANI_USERNAME}
+      - ANI_TOKEN=${PLEXANISYNC_ANI_TOKEN}
+      - INTERVAL=${PLEXANISYNC_INTERVAL}
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.yml:14]:
```suggestion
      - ANI_TOKEN=${PLEXANISYNC_ANI_TOKEN}
      - ANI_USERNAME=${PLEX_ANISYNC_ANI_USERNAME}
      - INTERVAL=${PLEXANISYNC_INTERVAL}
      - PGID=${PGID}
      - PLEX_SECTION=${PLEXANISYNC_PLEX_SECTION}
      - PLEX_TOKEN=${PLEXANISYNC_PLEX_TOKEN}
      - PLEX_URL=${PLEXANISYNC_PLEX_URL}
      - PUID=${PUID}
      - TZ=${TZ}
```

**Review by nemchik — CHANGES_REQUESTED** [review]

File: compose/.apps/plexanisync/plexanisync.yml (hunk 1 above)
Comment (nemchik) [reply compose/.apps/plexanisync/plexanisync.yml:5]: Added a review suggestion for it.

File: compose/.apps/plexanisync/plexanisync.x86_64.yml (hunk 4 above)
Comment (angrycuban13) [reply compose/.apps/plexanisync/plexanisync.x86_64.yml:3]: This is weird, looks like my commits didn't push last night

File: compose/.apps/plexanisync/plexanisync.aarch64.yml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+services:
+  plex:
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.aarch64.yml:2]:
```suggestion
  plexanisync:
```

File: compose/.apps/plexanisync/plexanisync.yml [hunk 7]
```diff
@@ -0,0 +1,23 @@
+services:
+  plexanisync:
+    command: /runsync.sh
+    container_name: plexanisync
+    environment:
+      - ANI_TOKEN=${PLEXANISYNC_ANI_TOKEN}
+      - ANI_USERNAME=${PLEX_ANISYNC_ANI_USERNAME}
```
Comment (nemchik) [inline compose/.apps/plexanisync/plexanisync.yml:7]:
```suggestion
      - ANI_USERNAME=${PLEXANISYNC_ANI_USERNAME}
```

**Review by nemchik — CHANGES_REQUESTED** [review]

**Review by nemchik — APPROVED** [review]

**Pull request merged**